ios/ReactNativeUsabilla/UsabillaInterface.swift

```diff
@@ -83,6 +83,25 @@ class UsabillaInterface: NSObject {
     func takeScreenshot(view: UIView) -> UIImage {
         return Usabilla.takeScreenshot(view)!
     }
+    
+    @objc(dismiss)
+    func dismiss() {
+        let _ = Usabilla.dismiss()
+    }
+    
+    @objc(getDefaultDataMasks)
+    func getDefaultDataMasks() -> [String] {
+        let str = Usabilla.defaultDataMasks
```

Comment:
Isn't assignment and return in two separate lines a bit verbose with such short line?
Just a personal preference, no need to take action on it